backend/src/main/java/com/cruru/evaluation/controller/dto/EvaluationResponse.java

```diff
@@ -1,11 +1,15 @@
 package com.cruru.evaluation.controller.dto;
 
+import java.time.LocalDateTime;
+
 public record EvaluationResponse(
         long evaluationId,
 
         int score,
 
-        String content
+        String content,
+
+        LocalDateTime createdDate
```

Comment:
```suggestion
        @JsonFormat(shape = Shape.STRING)
        LocalDateTime createdDate
```
이거 안붙이면 list로 간다고 하네요..

Reply:
도비와 함께 작업할 때, List[Year, Month, Day, ...] 이런 형태로 반환되서 붙였어요. 
냥인의 기능을 확인해보니 제대로 반환되네요 🧐


<img width="500" alt="image" src="https://github.com/user-attachments/assets/b2f09c5b-a2a0-46b4-92e8-9f7def53883a">

Reply:
@HyungHoKim00 @Chocochip101 
현재 저 어노테이션이 붙어있는 dto가 DashboardCreateRequest와 DashboardPreviewResponse인데, 이 dto들도 해당 어노테이션 없이 잘 반환되는 것 같아요.
이 코멘트 내용 반영할까요, 말까요?

Reply:
저는 안정성을 가져간다는 측면에서 붙이는게 좋아 보입니다!

Reply:
반영 완료입니다.